### PR TITLE
create_plugin: Don't forget about the plugin.json!

### PR DESCRIPTION
**Pull request by thopiekar**
An additional requirement since Cura 2.7.

### CODE REVIEW

File: create_plugin.py [hunk 1]
```diff
@@ -10,6 +10,10 @@ def checkValidPlugin(path):
     # A plugin must contain an __init__.py
     if not os.path.isfile(os.path.join(path, "__init__.py")):
         return False
+    
+    # A plugin must contain an __init__.py
```
Comment (ChrisTerBeke) [inline create_plugin.py]: This comment doen't seem to be relevant to `plugin.json` 😇

Comment (thopiekar) [reply create_plugin.py]: Ah, damn. Copy/pasted the section and forgot to correct the comment, too.
My finger was to early on the [commit] button :stuck_out_tongue_closed_eyes:

**Comment by thopiekar** [comment]
Corrected the comment in the code :wink:

**Comment by nallath** [comment]
CURA-4429

**Pull request merged**